core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -1430,6 +1430,47 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  // Send Change Notification for Workspace require WRITE access. Accept if OWNER or WRITE; Reject if READ or NO ACCESS
```

Comment:
Just making sure - even though we're putting the api for this under Workspaces, the tests should be moved to the notification.spec? Just want to make sure I'm understanding.